src/iframeWrapper.ts

```diff
@@ -61,31 +61,22 @@ export default class IframeWrapper {
   }
 
   public getSessionID = () => {
-    return window.localStorage.getItem(
+    const val = window.localStorage.getItem(
       `arcana-auth-${this.params.iframeUrl}-sessionID`
```

Comment:
Don't we need to also have app ID in the key?

Reply:
@mmjee the iframe url will have app id in it.

Reply:
Ah, that's OK then.